CollapseLauncher/Classes/CachesManagement/StarRail/Fetch.cs

```diff
@@ -6,10 +6,10 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using static CollapseLauncher.GameSettings.Base.SettingsBase;
```

Comment:
Sure, but it will be always the case since we sort it by alphabet so the `using static CollapseLauncher` will always be on top xD.
Joking aside, since this is just codeQA thing, IMO it's better to stick with just sorting it alphabetically, that way we can use automated CodeQA plugins like VS-internal or ReSharper for sorting all references

Reply:
Thanks for clarifying